apps/test/unit/templates/curriculumCatalog/curriculumCatalogCardTest.jsx

```diff
@@ -84,9 +87,14 @@ describe('CurriculumCatalogCard', () => {
   it('renders one topic, the first available from ordered list, if no subject present', () => {
     render(<CurriculumCatalogCard {...defaultProps} topics={topics} />);
 
-    screen.getByText(subjectsAndTopicsOrder[firstTopicIndexUsed], {
-      exact: false,
-    });
+    screen.getByText(
+      translatedCourseOfferingCsTopics[
```

Comment:
Wow this is much cleaner! We shouldn't need `exact: false` now –– it was relying on the one-word keys, but this is a much better solution

Reply:
Thanks to Turner for helping me figure this out!